tests/integration/security/testdata/requestauthn/global-jwt.yaml.tmpl

```diff
@@ -0,0 +1,37 @@
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
+metadata:
+  name: "default"
+  namespace: "{{ .RootNamespace }}"
+spec:
+  jwtRules:
```

Comment:
does this have to be namespace bound policy? can't we use label selector as well here? to be consistent with the authz.

Reply:
I actually want to keep it namespace/mesh wide to show it (JWT-alone) doesn't impact in-mesh traffic.